src/mp-units.cpp

```diff
@@ -0,0 +1,69 @@
+module;
+
+// core
+#include <gsl/gsl-lite.hpp>
+#include <mp-units/bits/external/hacks.h>
+#include <array>
+#include <compare>
+#include <concepts>
+#include <cstddef>
+#include <cstdint>
+#include <functional>
+#include <initializer_list>
+#include <iterator>
+#include <limits>
+#include <numbers>
+#include <numeric>
+#include <optional>
+#include <ranges>
+#include <string>
+#include <string_view>
+#include <tuple>
+#include <type_traits>
+#include <utility>
+
+// core-io
+#include <sstream>
+
+// core-fmt
+#include <mp-units/bits/fmt_hacks.h>
+
+// utility
+#include <chrono>
+#include <cmath>
+#include <random>
+
+export module mp_units;
+
+export
```

Comment:
Should we export everything (even the `detail` namespace) or things like `detail_unit` that explicitly states:

https://github.com/mpusz/mp-units/blob/a356db749ef8651d6b8c48cf732673286a0faa39/src/core/include/mp-units/unit.h#L242-L246

Reply:
Ideally, not.